bgpd/bgp_bmp.c

```diff
@@ -1016,14 +1310,16 @@ static struct stream *bmp_update(const struct prefix *p, struct prefix_rd *prd,
 	stream_putw(s, 0);
 
 	/* 5: Encode all the attributes, except MP_REACH_NLRI attr. */
-	total_attr_len = bgp_packet_attribute(NULL, peer, s, attr, &vecarr, NULL, afi, safi, peer,
-					      NULL, NULL, 0, 0, 0);
+	total_attr_len =
+		bgp_packet_attribute(NULL, peer, s, attr, &vecarr, NULL, afi,
+				     safi, peer, NULL, NULL, 0,
+				     safi != SAFI_MPLS_VPN, addpath_id);
 
```

Comment:
why are you limiting addpath to non mpls_vpn safi ?

can you document in some places, the remaining fixes to do ?

Reply:
addpath can be configured on L3VPN, I guess it is just a wireshark issue. This said, if you don't plan to do it, then you should note in a place (doc/developer/bmp.rst) that BMP L3VPN addpath is not yet supported.